src/mono/wasm/build/WasmApp.targets

```diff
@@ -140,7 +140,9 @@
 
       The path might not have been created yet, for example when creating a new project in VS, so don't use an Exists() check
     -->
-    <_AppBundleDirForRunCommand Condition="'$(_AppBundleDirForRunCommand)' == ''">$([System.IO.Path]::Combine($(OutputPath), 'browser-wasm', 'AppBundle'))</_AppBundleDirForRunCommand>
+    <_AppBundleDirForRunCommand Condition="'$(_AppBundleDirForRunCommand)' == '' and '$(TargetFramework)' != '' and $(OutputPath.Contains('browser-wasm'))">$([System.IO.Path]::Combine($(OutputPath), 'AppBundle'))</_AppBundleDirForRunCommand>
```

Comment:
The changes in these file are special casing for the library tests. But this targets file is shipped, and used by user projects. Instead of doing this we can set `<RunWorkingDirectory>$(BundleDir)</RunWorkingDirectory>` in `tests.wasm.targets`. We don't have the same issue in that file.